src/main/javascript/service/cat-validation-service.js

```diff
@@ -32,6 +32,31 @@ function CatValidationService($log,
     var that = this;
 
     /**
+     * Adds a field error to the given fieldErrors
+     * @param fieldName name of the faulty field
+     * @param errorMessage associated error message to display
+     * @param context affected context
+     * @private
+     */
+    var _addFieldError = function (fieldName, errorMessage, context) {
+        if (catMessagesConfig.knownFieldsActive === true) {
```

Comment:
please don't use `_` as a suffix for private stuff, this is unncessary, you wouldn't name your java variables that way either

also please just write
```js
function addFieldError(fieldName, errorMessage, context) {
/***/
}```

instead of assigning an anonymous function to a variable